### ISSUE TITLE
This message has to be translated

### ISSUE THREAD

**Issue by norohind** [EDCD/EDMarketConnector#1074]
https://github.com/EDCD/EDMarketConnector/blob/6ada384d1d885f54a33292fab9d487cd2cd6cee8/EDMarketConnector.py#L684

**Comment by Athanasius** [comment]
This is in the en.template file, which I'll make sure is the latest version on OneSky....

**Comment by Athanasius** [comment]
Added into the 5.1.0 translations.

**Comment by Athanasius** [comment]
You're quite right.  Looks like we need a generic 'Check' translation as well.

I've adjusted the code in `develop` and OneSky is processing the fresh upload.

**Comment by norohind** [comment]
1. Now only "Error" part is hardcoded when you already have common error message: https://github.com/EDCD/EDMarketConnector/blob/main/L10n/en.template#L190.
2. Not in all languages generic "Check" is suitable in this (Error: Check E:D journal file location) phrase, Russian one of them.

**Comment by Athanasius** [comment]
>     2. Not in all languages generic "Check" is suitable in this (Error: Check E:D journal file location) phrase, Russian one of them.

Yes, you're quite right.  I was being terribly mono-linguistic angolophone there.

We'll review all the translations this coming week and ensure they're all done in full for the best possible translations.  Stand by for that OneSky upload :) .

**Comment by norohind** [comment]
I don't see any such problems, feel free to close the issue.
